nncf/api/compression.py

```diff
@@ -58,7 +57,7 @@ def __call__(self, *args, **kwargs) -> Any:
         return self.calculate(*args, **kwargs)
 
 
-class CompressionScheduler:
+class CompressionScheduler(ABC):
```

Comment:
Do we really want to make this class abstract? Seems like it has the concrete implementation and I can not imagine other implementation.
@alexsu52 @vshampor What do you think?

Reply:
Leaving this as an interface won't hurt.

Reply:
Only for my understanding, what is the value of these changes?

Reply:
It was discussed offline. I understand arguments to use this way.